<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedOptionsDialog</class>
 <widget class="QDialog" name="AdvancedOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%1 - Advanced Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_extra_features">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Extra features</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_Maximizing">
          <property name="toolTip">
           <string>New analysis and gain functions can be activated</string>
          </property>
          <property name="text">
           <string>Enable &quot;Maximizing&quot; features</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ShowHiddenFields">
          <property name="toolTip">
           <string>Show &quot;Max Amplitude&quot; and dBGain fields in the list</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string>Show hidden fields</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_performance">
       <property name="title">
        <string>Performance</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_UseNoTempFiles">
          <property name="toolTip">
           <string>Gain modifications can be done directly in the files</string>
          </property>
          <property name="text">
           <string>Do not use Temp files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_ShowNoFileProgress">
          <property name="toolTip">
           <string>File progress bar can be switch off</string>
          </property>
          <property name="text">
           <string>Do not show file progress</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Hard core features</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Back end log depth</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="horizontalSlider_logBackendDepth">
          <property name="maximum">
           <number>2</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_logBackendDepth">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Trace log depth</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="horizontalSlider_logTraceDepth">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_logTraceDepth"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdvancedOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvancedOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_logBackendDepth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_logBackendDepth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_logTraceDepth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_logTraceDepth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_logBackendDepth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_logBackendDepth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_logTraceDepth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_logTraceDepth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
